### ISSUE TITLE
There is a problem with text in dialogs not being received

### ISSUE THREAD

**Issue by scripting** [scripting/feedlandSupport#163]
I've gotten a bunch of reports on this, if you have a dialog that's waiting for text to be entered, sometimes when you type into the dialog nothing happens. An example:

![image](https://user-images.githubusercontent.com/1686843/210364467-437c98fb-8d63-4c1f-bd2d-49a49e0afbe2.png)

I'm looking into it, I don't know what's causing it -- it must be a recent change. Will keep you posted here.

**Comment by scripting** [comment]
I believe I found and fixed this vexing problem.

Here's my theory of what was going on, for future reference.

1. The problem was introduced when I added keyboard navigation in the Feed List page.
2. In order for this to work, the Feed List code has to be listening to the keyboard.
3. However when there's a dialog in front of the Feed List, it should not be listening to the keyboard. It was.
4. The fix was to check to see if the "ask dialog" was in front when a key is pressed and if so, the Feed List ignores the keystroke.
5. Everyone is happy. :smile:

The reason I'm posting this here even though no one reported the problem here, is so that people who are listening can test this. 

Basically if you see the "ask dialog" freeze. It's used whenever FeedLand asks you to type a string. 

This gives you an idea of how tricky software can be, and how a new feature in one area can break a feature in a very distant part.

**Comment by mistersugar** [comment]
I am able to test the "ask dialog" but I am seeing a frozen dialog on the Search results page:

Starting on the "My feed list" page, I selected the Menu command "Search feeds ..." The search dialog appears, and I am able to type a word ('peace'). I clicked OK, and the Search results page appeared with 3 results. Immediately in the console I see an error (screenshot attached). I can put the cursor in the search field now anywhere in or after 'peace' but I can't delete or type additional characters.

<img width="557" alt="Screenshot 2023-01-03 at 8 43 11 PM" src="https://user-images.githubusercontent.com/3846301/210469459-88dd64e5-bb7e-4973-b15b-7c49713c6a8e.png">

**Comment by scripting** [comment]
@mistersugar -- i am able to reproduce that here. i'm going to look into fixing it now. ;-)

**Comment by scripting** [comment]
@mistersugar -- i believe i have this fixed now. you should be able to type into the text entry box on the search page, to search for something new.

**Comment by akaKenSmith** [comment]
It looks functional here, text entering and searches going through.
